secboot/secboot_tpm.go

```diff
@@ -85,6 +85,7 @@ func CheckKeySealingSupported() error {
 		logger.Noticef("%v", err)
 		return err
 	}
+	defer tpm.Close()
```

Comment:
do we care about returning the error here?

Reply:
Not specifically, and in all other operations we just defer it.